### PR TITLE
Added fire animation

### PR DESCRIPTION
**Pull request by seandk93**

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -7,11 +7,12 @@ import Jon from '@/components/Jon';
 import Pho from '@/components/Pho';
 
 import Header from '@/components/Header';
-
+import Weather from '@/components/Weather'
 Vue.use(Router);
 
 // Components
 Vue.component('header', Header);
+Vue.component('weather', Weather);
```
Comment (dkwl93) [inline src/router/index.js:15]: Is this needed here?

**Review by dkwl93 — CHANGES_REQUESTED** [review]

File: src/router/index.js (hunk 1 above)
Comment (dkwl93) [reply src/router/index.js:15]: otherwise, LGTM

Comment (seandk93) [reply src/router/index.js:15]: It's registering it as a global component. So you can use it as well

Comment (dkwl93) [reply src/router/index.js:15]: Oh okay, LGTM

**Review by dkwl93 — APPROVED** [review]

File: src/router/index.js (hunk 1 above)
Comment (seandk93) [reply src/router/index.js:15]: https://vuejs.org/v2/guide/components.html

**Pull request merged**